src/components/course-card/CourseCard.styles.ts

```diff
@@ -24,8 +24,9 @@ export const styles = {
     display: 'flex',
     flexDirection: 'column',
     justifyContent: 'space-between',
-    width: '390px',
-    minHeight: '245px',
+    minHeight: '285px',
+    minWidth: '360px',
```

Comment:
If you set `minWidth` for card. It's not looks perfect if we pass too long title for example or description as well. It'll break

<img width="955" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/42f77375-2089-4250-b0f2-1a8ce4333aac">

Reply:
I slightly changed grid so it doesn't break when there are very long titles or descriptions. Also, added `break-word` to title and description, so text can go to a new line. There is also added `height: 100%` for cards inside of this grid, so in case one card has bigger height (as result of double line title and description), other cards in the row have the same height too.
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/62070431/976f7622-6f77-45bb-b131-5293de93e2ce)

Reply:
> I slightly changed grid so it doesn't break when there are very long titles or descriptions. Also, added `break-word` to title and description, so text can go to a new line. There is also added `height: 100%` for cards inside of this grid, so in case one card has bigger height (as result of double line title and description), other cards in the row have the same height too. ![image](https://private-user-images.githubusercontent.com/62070431/322647027-976f7622-6f77-45bb-b131-5293de93e2ce.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyNTI3MzMsIm5iZiI6MTcxMzI1MjQzMywicGF0aCI6Ii82MjA3MDQzMS8zMjI2NDcwMjctOTc2Zjc2MjItNmY3Ny00NWJiLWIxMzEtNTI5M2RlOTNlMmNlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDA3MjcxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4MWZhOWUyNGM5ZTk3NTQ4MjMzNjUwMWQ5MGE5MzNhOTY4NzgwYWMxZjEwZWQxYzQ3ZmE1MWI0NzNkMGRlNzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.eMQwhd29ZGMnMLEmR8HgcHkrRgOq-dEUK6rI6cTdgAk)

I think that you can try to write `wordBreak` property only once for your `card` style. 
And not repeat it for `title` and `description` and also can write `height` property and not to create another one

Reply:
Thanks for suggestion. I moved `wordBreak` to general card style in recent push. Also moved `height: 100%` directly to card styles, instead of setting it in `MyCoursesCardsList`.